getuto

```diff
@@ -159,14 +159,19 @@ if [[ ! -d ${GNUPGHOME} ]] ; then
 
 	# TODO: keys.openpgp.org lacks a UID for our keys, need to verify email
 	for keyserver in "${mykeyservers[@]}" ; do
-		gpg ${QUIET_GPG} --batch --keyserver "${keyserver}" --recv-keys ${myrelkeys} || true
+		einfo "Importing release engineering keys from: ${keyserver}"
```

Comment:
This is adding unconditional status messages. Currently getuto only prints the major action being done (initializing, or updating, the keyring as a whole). I'm of two minds about this change, but either way, it deserves to be a distinct commit with distinct rationale, not "do it in parallel".